<section id="pubs" class="bg-light-gray">
    <div class="container">
        <div class="row">
            <div class="col-lg-12 text-center">
                <h2 class="section-heading">Publications</h2>
            </div>
        </div>

        <div class="subtitle">In Preparation</div>
        <!-- Pub1 -->
        <div class="block">
            <div class="abstract">
                <img class="project_img" src="img/plot-pfas.png">
                <div class="right">
                    <div class="research_title">
                        Mapping PFAS in Groundwater: Statistical Mapping of PFAS in Groundwater Throughout the
                        Contiguous United States
                    </div>
                    <p>
                        Per-and polyfluoroalkyl substances (PFAS) are synthetic chemicals that are increasingly being
                        detected in groundwater. The negative health consequences associated with human exposure to PFAS
                        make it essential to quantify the distribution of PFAS in groundwater systems. Mapping PFAS
                        distributions is particularly challenging because a national patchwork of testing and reporting
                        requirements has resulted in sparse and spatially biased data. In this analysis, an
                        Inhomogeneous Poisson Process (IPP) modeling approach is adopted from ecological statistics to
                        continuously map PFAS distributions in groundwater across the contiguous United States. The
                        model is trained on a unique dataset of 8,910 PFAS groundwater measurements, using combined
                        concentrations of two PFAS analytes. The IPP model predictions are compared with results from
                        random forest models to highlight the robustness of this statistical modeling approach on sparse
                        datasets. This analysis provides a new approach to not only map PFAS contamination in
                        groundwater but also prioritize future sampling efforts.
                    </p>
                </div>
            </div>
            <div class="publication">
                <span class="emph">B. Park</span>,
                <a>H. Kang</a>, and
                <a>C. Zahasky</a>.
                <br>
                <!-- eSVD-DE: Cohort-wide differential expression in single-cell RNA-seq data using exponential-family embeddings<br>
                              <a href="https://bmcbioinformatics.biomedcentral.com/" target="_blank">BMC Bioinformatics</a>
                To appear (2024):
                (<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-024-05724-7" target="_blank">link</a>)
                             (<a href="https://www.biorxiv.org/content/10.1101/2023.11.22.568369v1" target="_blank">biorxiv</a>)
                             (<a href="https://github.com/linnykos/eSVD2" target="_blank">code: method</a>)
                             (<a href="https://github.com/linnykos/eSVD2_examples" target="_blank">code: analysis</a>) -->
            </div>
        </div>

        <!-- <div class="subtitle">Published </div> -->
    </div>